doc/pages/services.json

```diff
@@ -98,7 +98,19 @@
   {
     "name":"US Department of Veteran's Affairs | Mental Health",
     "link":"https://www.mentalhealth.va.gov/gethelp.asp",
-    "tags":["free", "suicide prevention", "homeless", "directory", "veterans"],
+    "tags":["free", "suicide prevention", "homeless", "directory", "veterans", "government"],
     "languages":["en"]
+  },
+  {
+    "name":"Resource Links - Disabled Veterans National Foundation",
```

Comment:
Oops I forgot to add (US) to the end here. Should I correct that? And also the link above (but it starts with "US").

Reply:
Maybe rephrase to "Disabled Veterans National Foundation Resources (US)"?